# message for displaying artifact images, with some meta-data (num images artifact has, etc.)

sensor_msgs/Image img
uint8 image_ind
uint8 num_images
float32[] pose